.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Setup miniconda & zarp env
         uses: conda-incubator/setup-miniconda@v2
         with:
-          python-version: "3.7.4"
+          python-version: "3.10"
```

Comment:
These changes are rather addressing #108 (and the discussion therein), not #122

Reply:
I am in the process of adding the documentation. Also the issue you refer to is #114 as the #122 is the PR.